samples/MegaDrone/CMakeLists.txt

```diff
@@ -30,4 +30,3 @@ include_directories (${OBOE_DIR}/include)
 # Append compiler optimization flags to both the debug and release build variants
 set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Ofast")
 set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -Ofast")
```

Comment:
Should we change these to the new technique?

Reply:
+1